pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -253,6 +252,34 @@ func (w *watcher) modifyFiles(latestEvent *model.Event, eventCfg *config.EventWa
 	}, nil
 }
 
+// modifyYAML returns a new YAML content as a first returned value if the value
+// of given field was outdated. Nil means it's already up-to-date.
+func modifyYAML(path, field, latestData string) ([]byte, error) {
```

Comment:
nit: "latestData" -> "newValue"

Reply:
Maybe instead of using nil to express the "up-to-date" status, we should have another return value with boolean type.
`[]byte, bool, error`

Reply:
That is so true. Let me do so.